autogtp/Game.cpp

```diff
@@ -317,6 +317,16 @@ bool Game::dumpTraining() {
     return true;
 }
 
+bool Game::dumpDebug() {
+    QTextStream(stdout) << "Dumping " << m_fileName + ".debug.txt" << endl;
+
+    if (!sendGtpCommand(qPrintable("dump_debug " +
+                        m_fileName + ".debug.txt"))) {
```

Comment:
Did you consider replacing this with `return sendGtpCommand(...);`?

Reply:
I kept the same style as the other functions around it. I leave it to the original author @marcocalignano to decide if he wants to change the style.

Reply:
@killerducky  Change it to return please.

Reply:
I'll pull this next then.

Reply:
fixed.